oper8/verify_resources.py

```diff
@@ -282,3 +289,9 @@ def is_expected_reason() -> bool:
         return obj_reason == expected_reason
 
     return is_expected_status() and is_expected_reason()
+    return is_expected_status() and is_expected_reason()
```

Comment:
What happened here??

Reply:
VS Code likes to autoformat things :( I think it removed a newline?

Reply:
This diff shows six copies of the same return statement?